CODE_OF_CONDUCT.md

```diff
@@ -2,6 +2,8 @@
 
 Be respectful toward other members of the community. Don't initiate or engage in discussions that are designed to insult, inflame, attack, or incite hate/discrimination/bullying against others. Don't feed the trolls. If we detect any kind of behaviour we consider unacceptable, we reserve the right to ban, block, or remove the involved members from any or all of our communities.
 
+Never ever attempt to contribute or even look at leaked code from Microsoft Windows or any other proprietary software. Looking at but not copying proprietary code with expired copyright may or may not be acceptable depending on the specific artifact, and will be resolved on case-by-case basis.
```

Comment:
```suggestion
Never ever contribute or look at leaked code from Microsoft Windows or any other proprietary software. Looking at code protected by copyright is not acceptable. If you have seen Microsoft Windows source code, you are not allowed to contribute to this project.
```

Reply:
Prohibition of looking at out-of-copyright proprietary code except for case-by-case exceptions is intended, not a mistake.

Reply:
It is to prevent mass copying of code from a leak of, say, Windows happens to go out of copyright before it is officially open sourced.

Reply:
> Looking at code protected by copyright is not acceptable.

This does not mean what you think it does. GPL code is protected why copyright -- in fact, that's the only reason the GPL works.

Reply:
Fair criticism.

Reply:
I would remove the 'attempt' from this sentence, because if you want to be anal you can say 'I did not attempt to do it, I did do it.'

Reply:
I think the phrase I was looking for was 'prohibited by copyright' rather than 'protected by copyright'